### PR TITLE
elpicpie.d: publics above the fold, privates below

### PR DESCRIPTION
**Pull request by WalterBright**
Just a cut&paste job.

### CODE REVIEW

**Review by JinShil — APPROVED** [review]
The diff looks like hell, but I trust Walter.

File: src/dmd/backend/elpicpie.d [hunk 1]
```diff
@@ -478,531 +826,182 @@ private elem *el_picvar(Symbol *s)
...
+        switch (s.Sclass)
         {
-            /* Cannot access address of code from code.
-             * Instead, create a data variable, put the address of the
-             * code in that data variable, and return the elem for
-             * that data variable.
-             */
-            Symbol *sd = symboldata(Offset(DATA), TYnptr);
-            sd.Sseg = DATA;
-            Obj.data_start(sd, _tysize[TYnptr], DATA);
```
Comment (rainers) [inline src/dmd/backend/elpicpie.d]: This line was removed. Is this deliberate?

Comment (WalterBright) [reply src/dmd/backend/elpicpie.d]: No. Good catch!

Comment (WalterBright) [reply src/dmd/backend/elpicpie.d]: And even if it was deliberate, I'd never do that for a refactoring advertised as a cut&paste job. It'd be a separate PR.

**Review by rainers — APPROVED** [review]

**Pull request merged**